app/models/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,11 @@
+module CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def remote_delete
+    raw_delete
+  end
+
+  def raw_delete
```

Comment:
@durandom @agrare I shall go with: 
```ruby
def cloud_object_store_container_delete
   raw_delete
end
```
There will be place to drop policies in `cloud_object_store_container_delete` in the future, but I would not include them in this PR.

Are you OK with this?